pkgs/servers/adguardhome/version.nix

```diff
@@ -0,0 +1 @@
+"0.106.3"
```

Comment:
Can we remove this extra file and sed the version in the default.nix?

Reply:
Would be nice, will get to it.